pybamm/models/submodels/tortuosity/bruggeman_tortuosity.py

```diff
@@ -0,0 +1,35 @@
+#
+# Class for Bruggeman tortuosity
+#
+import pybamm
+
+from .base_tortuosity import BaseModel
+
+
+class Bruggeman(BaseModel):
+    """Submodel for Bruggeman tortuosity
+
+    **Extends:** :class:`pybamm.tortuosity.BaseModel`
+    """
+
+    def __init__(self, param, phase, set_leading_order=False):
+        super().__init__(param, phase)
+        self.set_leading_order = set_leading_order
+
+    def get_coupled_variables(self, variables):
+        param = self.param
+
+        if self.phase == "Electrolyte":
+            eps = variables["Porosity"]
+        elif self.phase == "Electrode":
+            eps = variables["Active material volume fraction"]
+
+        eps_n, eps_s, eps_p = eps.orphans
+        tor = pybamm.Concatenation(
+            eps_n ** param.b_n, eps_s ** param.b_s, eps_p ** param.b_p
```

Comment:
If I understand it correctly, here all the parameters `param.b_j` are the same for electrode and electrolyte in a given part. I think that is not necessarily true because the geometry that defines the tortuosity is quite different between the electrode and electrolyte. Would it be possible to pass each `b_j` as an argument when you call a Bruggeman tortuosity model?

Reply:
Would it just be a case of having a different parameter for electrolyte and solid?

Reply:
Yes. Also, they could be different for different transport phenomena in the same domain. Right now it is not a problem as we only need one number for electrode and one for electrolyte, but later on we might need more.